moon-store-mac-os/moon-store-mac-os/Networking/Models/UserRegisterModel.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  UserRegisterModel.swift
+//  moon-store-mac-os
+//
+//  Created by Jose Luna on 1/7/25.
+//
+
+struct UserRegisterModel: Codable {
+    var firstName: String = ""
+    var lastName: String = ""
+    var identification: String = ""
+    var phone: String = ""
+    var address: String = ""
+    var email: String = ""
+    var password: String = ""
+    var confirmPassword: String = ""
+    
+    func cannotRegisterYet() -> Bool {
```

Comment:
Este metodo podría ser una variable computada, aparte el objetivo podría cambiar a validar si puede ser creado

```suggestion
    var canBeRegister: Bool {
```